contributors/guide/pull-requests.md

```diff
@@ -622,6 +623,18 @@ reference to this documentation if they come to the conclusion that the
 difficulty of properly reviewing the Pull Request outweighs the benefit that
 the Pull Request provides.
 
+## AI Guidance
+
+Using AI tools to help write your PR is acceptable, but as the author, you are responsible for understanding every change. 
+
+Reviewers may ask questions about your AI-assisted code, and if you cannot explain why a change was made, the PR will be closed.
+
+When responding to review comments, please do so without relying on AI tools. Reviewers want to engage directly with you, not with generated responses.
+
+If you used AI tools in preparing your PR, please disclose this in the "Special notes for your reviewer" section.
+
+Commit messages should follow the [Kubernetes policy](#commit-message-guidelines). AI generated commit messages are discouraged.
```

Comment:
Nit: can you squash all of these sentences into a single paragraph?

Reply:
Updated both.